tests/legacy/unit-tests/admin/class-wc-tests-admin-duplicate-product.php

```diff
@@ -96,6 +96,27 @@ public function test_variable_product_duplication_unique_slug_generation() {
 		}
 	}
 
+	/**
+	 * Tests that the filter will exclude metadata from the duplicate as-expected.
+	 */
+	public function test_filter_allows_excluding_metadata_from_duplicate() {
```

Comment:
Not a blocker, but https://protonpower.wordpress.com/2020/06/09/woocommerce-core-test-suite-structure says:

> No new tests should be written in the legacy folder, even if a corresponding test class is already present there.

That was written by some @ObliviousHarmony who I think you pretty much trust 😁

Reply:
I've moved the test file to its new home!